Lib/locale.py

```diff
@@ -321,6 +327,11 @@ def delocalize(string):
         string = string.replace(dd, '.')
     return string
 
+def localize(string, grouping=False, monetary=False):
+    """Parses a string as locale number according to the locale settings."""
+    float(string)
```

Comment:
What's the float(string) for?

Reply:
Ensure that it is a string that represent a valid number.